packages/@secretlint/config-loader/src/index.ts

```diff
@@ -8,9 +8,18 @@ import {
 } from "@secretlint/types";
 import { secretLintProfiler } from "@secretlint/profiler";
 import { SecretLintModuleResolver } from "./SecretLintModuleResolver";
-import { moduleInterop } from "@textlint/module-interop";
 import { validateConfig, validateRawConfig } from "@secretlint/config-validator";
 
+export function moduleInterop<T>(moduleExports: T): T {
+    if (moduleExports && (moduleExports as any).__esModule) {
+        return (moduleExports as any).default?.default!;
```

Comment:
`module.exports.default = 1`

is `default.default`.
`__esModule` is broken.
<img width="1882" alt="image" src="https://user-images.githubusercontent.com/19714/132990630-6091aa6e-ab32-4be9-962b-418133226200.png">